packages/vm/tests/tester/config.ts

```diff
@@ -375,14 +375,12 @@ export function getCommon(network: string): Common {
           | null
```

Comment:
Let's add `undefined` here also?

Reply:
It's already covered by the `forkBlockNumber ?? 0` option below methinks.  We can make it explicit though for future reference.

Reply:
Yeah, I think making it explicit is nice, avoids someone else running into a similar issue eventually